hub.Dockerfile

```diff
@@ -1,10 +1,12 @@
 # Base image
-FROM node:8.12.0-slim as base
```

Comment:
hub.Dockerfile is no longer needed right?



hub.Dockerfile is no longer needed, you can delete